deltachat-ios.xcodeproj/project.pbxproj

```diff
@@ -954,6 +969,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
+				"HEADER_SEARCH_PATHS[arch=*]" = /Users/nayooti/iOSDeveloper/rpgp_temp;
```

Comment:
this is not going to make things easier for others

Reply:
Didn't know my absolute path will show up in the project-file.

I had to do this, because since the last release (update code and rpgp) the project does not compile anymore due to a missing rgpg-header file.

Reply:
If you run `make install` in the `rpgp/pgp-ffi` directory, it should place the header in an accessible location. Alternatively maybe we should simply check it in, and use a project local version of it.

Reply:
Where exactly should I run `make install`? 
In my case `deltachat-ios/deltachat-ios/libraries/rpgp` does not contain a `pgp-ffi`-file. Only librgpg.a is listed.

Reply:
```
> git clone https://github.com/rpgp/rpgp
> cd rpgp/pgp-ffi
> make install
```

should do the trick

Reply:
but really we should copy the header file into the project, I can do that in a bit

Reply:
that would be nice. tx